public/app/features/gops/configuration-tracker/hooks/irmHooks.ts

```diff
@@ -0,0 +1,380 @@
+            button: {
+              type: 'openLink',
+              urlLink: {
+                url: '/alerting/notifications/receivers/new',
+              },
+              label: 'Connect',
+              done: isOnCallContactPointReady(contactPoints),
+              urlLinkOnDone: {
+                url: '/alerting/notifications',
+              },
+              labelOnDone: 'View',
+            },
+          },
+          {
+            title: 'Create alert rule',
+            description: 'Create an alert rule to monitor your system.',
+            button: {
+              type: 'openLink',
+              urlLink: {
+                url: '/alerting/new',
+              },
+              label: 'Create',
+              urlLinkOnDone: {
+                url: '/alerting/list',
+              },
+              labelOnDone: 'View',
+              done: isCreateAlertRuleDone(),
+            },
+          },
+        ],
+      },
+      {
+        title: 'Respond',
+        description: 'Configure OnCall and Incident',
+        steps: [
+          {
+            title: 'Initialize Incident plugin',
+            description: 'Initialize the Incident plugin to declare and manage incidents.',
+            button: {
+              type: 'openLink',
+              urlLink: {
+                url: '/a/grafana-incident-app/walkthrough/generate-key',
+              },
+              label: 'Initialize',
+              urlLinkOnDone: {
+                url: '/a/grafana-incident-app',
+              },
+              labelOnDone: 'View',
+              done: incidentPluginConfig?.isInstalled,
+            },
+          },
+          {
+            title: 'Connect your Messaging workspace to OnCall',
+            description: 'Receive alerts and oncall notifications within your chat environment.',
+            button: {
+              type: 'openLink',
+              urlLink: {
+                url: '/a/grafana-oncall-app/settings',
+                queryParams: { tab: 'ChatOps', chatOpsTab: 'Slack' },
+              },
+              label: 'Connect',
+              urlLinkOnDone: {
+                url: '/a/grafana-oncall-app/settings',
+                queryParams: { tab: 'ChatOps' },
+              },
+              labelOnDone: 'View',
+              done: chatOpsConnections.is_chatops_connected,
+            },
+          },
+          {
+            title: 'Connect your Messaging workspace to Incident',
+            description:
+              'Automatically create an incident channel and manage incidents directly within your chat environment.',
+            button: {
+              type: 'openLink',
+              urlLink: {
+                url: '/a/grafana-incident-app/integrations/grate.slack',
+              },
+              label: 'Connect',
+              urlLinkOnDone: {
+                url: '/a/grafana-incident-app/integrations',
+              },
+              done: incidentPluginConfig?.isChatOpsInstalled,
```

Comment:
Shouldn't the `done` property belong to the `step` rather than to the button?

Reply:
good point. Let's update it